### PR TITLE
Add support for VIP, Moderator and Broadcaster AccessState

### LINKED ISSUE #133: Add Moderator, VIP and Broadcaster to the list of Access State
Is it possible to have `VIP` , `Moderator` and `Broadcaster` on this list?

### PR DESCRIPTION
**Pull request by jwalter**
Closes #133

### CODE REVIEW

**Comment by parithon** [comment]
Thanks for the PR @jwalter. Would it be possible to rebase this on the Comfy branch? We think these changes will fit much better into the newer Comfy system as a lot of the badges and other Twitch related branding is handled much easier with that package. Additionally, please change the PR to target the Comfy branch once you rebase as well.

**Comment by jwalter** [comment]
Hi @parithon , not sure I managed to do this the way you wanted. History looks a bit messy?

**Comment by parithon** [comment]
@jwalter I cleaned up the git history... all the tests passed (THANK YOU)! Can you verify I didn't mess anything up along the way? Thanks for your effort!

File: package.json [hunk 1]
```diff
@@ -170,7 +173,7 @@
     "@types/node": "^10.12.21",
     "@types/node-fetch": "^2.1.7",
     "@types/sinon": "^7.0.7",
-    "@types/tmi.js": "~1.4.0",
+    "@types/tmi.js": "~1.4.2",
```
Comment (parithon) [inline package.json:176]: I'm not sure we even need these types anymore since we are using Comfy. However, that might be outside the scope of this PR.

**Review by parithon — APPROVED** [review]

**Review by parithon — APPROVED** [review]

**Comment by MichaelJolley** [comment]
@all-contributors please add @jwalter for code and tests

**Pull request merged**